README.md

```diff
@@ -58,6 +58,8 @@ CQ-editor is an IDE that allows users to edit CadQuery model scripts in a GUI en
 * A CadQuery stack inspector.
 * Export to various formats, including STEP and STL, directly from the menu.
 
+***PLEASE NOTE:*** If you are planning to use CQ-editor, it requires a version of Anaconda which includes Python 3.7. A future release of CQ-editor will update it to be compatible with Python 3.8.
```

Comment:
`it requires a conda env with python 3.7`

Reply:
I might add a short note on setting up a conda env.

Reply:
Yeah, that's an important distinction between the env and Anaconda itself. I'll make that change.

Reply:
Likewise, the README has been updated.